pages/package/[...packageString]/components/TreemapSection.tsx

```diff
@@ -156,11 +164,11 @@ class TreemapSection extends Component {
                     className="treemap__percent"
                     style={{
                       fontSize: `${
-                        14 + Math.min(dep.percentShare * 1.2, 25)
+                        14 + Math.min((dep.percentShare || 0) * 1.2, 25)
                       }px`,
                     }}
                   >
-                    {dep.percentShare.toFixed(1)}
+                    {(dep.percentShare || 0).toFixed(1)}
```

Comment:
This looks rather verbose. We should ensure that if `percentShare` is nullable, the right fallback value is set on the API side.